frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -669,56 +678,60 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
       dataTest="dropdown-selectbox"
     />
   ) : (
-    <div className="form-group">
-      <RadioInput
-        onChange={() => {
-          setUseSuggestedNSForSingleInstallMode(true);
-          setTargetNamespace(operatorSuggestedNamespace);
-        }}
-        value={operatorSuggestedNamespace}
-        checked={useSuggestedNSForSingleInstallMode}
-        title={t('olm~Operator recommended Namespace:')}
-      >
-        {' '}
-        <ResourceIcon kind="Project" />
-        <b>{operatorSuggestedNamespace}</b>
-      </RadioInput>
-      {useSuggestedNSForSingleInstallMode && suggestedNamespaceDetails}
-      <RadioInput
-        onChange={() => {
-          setUseSuggestedNSForSingleInstallMode(false);
-          setTargetNamespace(null);
-        }}
-        value={operatorSuggestedNamespace}
-        checked={!useSuggestedNSForSingleInstallMode}
-        title={t('olm~Select a Namespace')}
-      />
-      {!useSuggestedNSForSingleInstallMode && (
-        <>
-          <NsDropdown
-            id="dropdown-selectbox"
-            selectedKey={selectedTargetNamespace}
-            onChange={(ns) => setTargetNamespace(ns)}
-            dataTest="dropdown-selectbox"
-          />
-
-          {suggestedNamespace !== selectedTargetNamespace && (
-            <Alert
-              isInline
-              className="co-alert pf-c-alert--top-margin"
-              variant="warning"
-              title={t(
-                'olm~Not installing the Operator into the recommended namespace can cause unexpected behavior.',
-              )}
+    <>
+      <div className="form-group">
+        <RadioInput
```

Comment:
Take 2 with this comment:  Rather than duplicating these components from about, make them in to constants so there is a single source for both.

Reply:
updated